sales.html

```diff
@@ -3,9 +3,10 @@
   <head>
     <meta charset="utf-8">
     <title></title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="salesPage.css">
```

Comment:
try to reserve camelCase for Javascript, I would recommend snake_case or kebab-case for file names, id's etc.